StellaNowSDKDemo/src/Program.cs

```diff
@@ -61,6 +61,12 @@ static async Task Main(string[] args)
             stellaSdk.SendMessage(message);
             logger.LogInformation("New Message Queued");
 
+            logger.LogInformation(
+                "Queue has messages {HasMessages} and count is {Count}",
+                stellaSdk.HasMessagesPendingForDispatch(),
+                stellaSdk.MessagesPendingForDispatchCount()
+            );
+            
```

Comment:
maybe as we have example program now, we should make use of `HasMessagesPendingForDispatch()` method and add a loop around `// Disconnect and terminate` and wait for all messages to be transmitted before we kill?

Reply:
ok I see it's done in https://github.com/stella-systems/stellanow-sdk-csharp/pull/7/files, right?